unittests/IR/ValueHandleTest.cpp

```diff
@@ -34,8 +34,8 @@ class ConcreteCallbackVH : public CallbackVH {
   ConcreteCallbackVH(Value *V) : CallbackVH(V) {}
 };
 
-TEST_F(ValueHandle, WeakVH_BasicOperation) {
-  WeakVH WVH(BitcastV.get());
+TEST_F(ValueHandle, WeakTrackingVH_BasicOperation) {
```

Comment:
Do we run these tests as part of the dxc unit tests?

Reply:
Doesn't seem like it :/ The test seems like a very nice .exe that we can just run though.

Reply:
They're all passing right now, which is good. I'll open a separate issue to run them.